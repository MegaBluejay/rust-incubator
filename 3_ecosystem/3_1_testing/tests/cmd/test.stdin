1000000000000000
50
-1
0
8
