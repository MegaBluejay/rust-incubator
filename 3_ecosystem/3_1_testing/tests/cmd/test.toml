bin.name = "step_3_1"
args = ["8"]
