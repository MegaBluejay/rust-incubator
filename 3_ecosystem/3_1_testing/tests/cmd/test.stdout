Guess the number!
Please input your guess.
Please input your guess.
You guessed: 50
Too big!
Please input your guess.
Please input your guess.
You guessed: 0
Too small!
Please input your guess.
You guessed: 8
You win!
